cmd/vic-machine/common/debug.go

```diff
@@ -14,19 +14,23 @@
 
 package common
 
-import "gopkg.in/urfave/cli.v1"
+import (
+	"gopkg.in/urfave/cli.v1"
+
+	"github.com/vmware/vic/pkg/flags"
+)
 
 type Debug struct {
-	Debug int `cmd:"debug"`
+	Debug *int `cmd:"debug"`
 }
 
 func (d *Debug) DebugFlags() []cli.Flag {
 	return []cli.Flag{
-		cli.IntFlag{
-			Name:        "debug, v",
-			Destination: &d.Debug,
-			Usage:       "[0(default),1...n], 0 is disabled, 1 is enabled, >= 1 may alter behaviour",
-			Hidden:      true,
+		cli.GenericFlag{
+			Name:   "debug, v",
```

Comment:
I thought `--debug` was already an option flag, so why do we need to change this?